java/apitest/src/test/java/e2e/registry/registry.feature

```diff
@@ -223,7 +223,7 @@ Feature: Registry api tests
     When method post
     Then status 200
     And response.result.Teacher.transactionId.length > 0
-    * sleep(3000)
+    * sleep(7000)
```

Comment:
Why is this changed.

Reply:
This happened due to one of the rebase probably, undid all these changes.